examples/mobile/UIComponents/components/controllers/index.html

```diff
@@ -62,6 +62,11 @@ <h1>
 						On/Off switch
 					</a>
 				</li>
+				<li class="ui-li-anchor">
+					<a href="adddeletebutton.html">
```

Comment:
Could you change the name of file as `add-delete-button.html` ?

Reply:
Done!